build/buildBuildImages.sh

```diff
@@ -89,18 +96,23 @@ function buildDockerImage() {
 	echo
 	echo Building a base image for tests...
 	# Do not write this image tag to the artifacts file as we do not intend to push it
```

Comment:
I supposed this comment is no longer valid?

Reply:
I don't think that we need to use the `oryxtests` registry anymore, but it's still being used in a lot of tests -- I'll probably create a separate PR after this one gets merged to convert these tests to using the new environment variable pattern so this PR isn't cluttered with this work.